static/bidder-info/trustx.yaml

```diff
@@ -1,17 +1,4 @@
-endpoint: "https://grid.bidswitch.net/sp_bid?sp=trustx"
+aliasOf: grid
 maintainer:
   email: "grid-tech@themediagrid.com"
 gvlVendorID: 686
-capabilities:
-  app:
-    mediaTypes:
-      - banner
-      - video
-  site:
-    mediaTypes:
-      - banner
-      - video
-userSync:
```

Comment:
There is a subtle difference with removing the usersync config from this adapter. The trustx identity pool will be ignored in favor of grid. We might want to think a little more about this before making a change. Please revert the usersync section to bring back the previous behavior.